deposit-to-exchange.md

```diff
@@ -1 +1,138 @@
-# Title
+# Confirm a Deposit to an Exchange
+
+Deposits are made in several stages. 
+One of the stages is to go through the [Exchange server](https://docs.flare.network/exchange/architecture/#architecture-of-an-exchange), which monitors for deposits and updates the Balances database. 
+Run the code below to have the Exchange server listen for deposit transactions and confirm the deposits.
+
+On a blockchain, multiple [validators](h[ttps://docs.flare.network/tech/glossary/](https://docs.flare.network/tech/validators/)) examine newly created transactions and come to an agreement about whether they are valid. 
+When they are valid, they are added to a new block on the chain and identified by a one-way cryptographic algorithm, called a _hash_. 
+The one-way hash prevents transactions from being reverted. 
+[Blocks](https://docs.flare.network/tech/glossary/) are added to the [blockchain](https://docs.flare.network/tech/glossary/) in chronological order and it may take a few blocks for the hash to be completed.
+To prevent problems, we confirm deposits several blocks back (five blocks in our code), when the blocks are most likely to be irreversible.
+
+<!-- The [Architecture of an Exchange doc](https://docs.flare.network/exchange/architecture/) says, "To avoid problems, the Exchange should only act on transactions appearing on blocks old enough for the chance of them being reverted to be negligible."  Is this doc referring to the fact that blocks are irreversible except in the case of a severe attack when an ecosystem might agree to fork. Am I on the right track? 
+I have never heard of a period where they could possibly be reverted, so I'm imagining that is the brief moment waiting for the cryptographic hash to be in place, yes?  -->
+
+Newly submitted deposits to our receiving address are labeled "pending."
+After five blocks have been created, valid deposits are labeled "confirmed."
+
+Run the javascript below to see pending and confirmed transactions and which block number they were added to.
+
```

Comment:
I don't think this article should be a primer on blockchain, so I limited blockchain explanation to only what they need to know for this procedure. Ideally, we would have high-level blockchain concepts so we don't have to repeat the concepts for every article that they apply to. 
 The existing glossary definitions are part of that. 
 Here are a few more definitions that I recommend adding to the glossary: 

 | Term | Description |
 | ----------- | ----------- |
 | address | The address of a wallet where assets can be stored.
 Represented by a hash. |
 | block header | Data about the block, including a timestamp of when the block was a created, a hash representing the address of the previous block, and a Merkle root hash (a cryptographic hash of all of the transactions included in the block) |
 | block number | Blocks are numbered sequentially in chronological order of creation |
 | hash | An output of a string of a fixed size derived from the transformation of data of any length. 
 `tx.hash` is the hash of the transaction.
 It is generated by a one-way cryptographic algorithm, for which the original data cannot be retrieved by decryption.
 Hashing stores passwords and prevents fraudulent transactions and double spending in blockchain. |
 | node | A programmed participant in the network that can interact with other nodes.
 Each node has different responsibilities, such as client or validator. |